VERSION "Harness Resistance Demo" 
NS_ : 
NS_DESC_ 
CM_ 
BA_DEF_ 
BA_ 
VAL_ 
CAT_DEF_ 
    CAT_ 
    FILTER 
    BA_DEF_DEF_ 
    EV_DATA_ 
    ENVVAR_DATA_ 
    SGTYPE_ 
    SGTYPE_VAL_ 
    BA_DEF_SGTYPE_ 
    BA_SGTYPE_ 
    SIG_TYPE_REF_ 
    VAL_TABLE_ 
    SIG_GROUP_ 
    SIG_VALTYPE_ 
    SIGTYPE_VALTYPE_ 
    BO_TX_BU_ 
    BA_DEF_REL_ 
    BA_REL_ 
    BA_DEF_DEF_REL_ 
    BU_SG_REL_ 
    BU_EV_REL_ 
    BU_BO_REL_ 
    SG_MUL_VAL_ 
 
BS_: 
 
BU_ ECU_A ECU_B DCDC 
 
BO_ 273 ECU_A_STATUS: 8 ECU_A 
 SG_ ECUA_Supply_Voltage : 0|16@1+ (0.1,0) [0|40] "V"  ECU_A 
 
BO_ 274 ECU_B_STATUS: 8 ECU_B 
 SG_ ECUB_Supply_Voltage : 0|16@1+ (0.1,0) [0|40] "V"  ECU_B 
 
BO_ 275 DCDC_STATUS: 8 DCDC 
 SG_ DCDC_Output_Voltage : 0|16@1+ (0.1,0) [0|40] "V"  DCDC 